src/Angular.js

```diff
@@ -1602,6 +1622,11 @@ function angularInit(element, bootstrap) {
     }
   });
   if (appElement) {
+    if (!isAutoBootstrapAllowed) {
```

Comment:
@mmccallum FWIW, when auto bootstrap gets disabled, Angular prints out a pretty clear message in the console.